contracts/foundry.toml

```diff
@@ -19,6 +19,7 @@ fs_permissions = [
 
 [fuzz]
 seed = "0x6eed"
+runs = 10
```

Comment:
just 10?

Reply:
Did this cause fork tests were very slow let me change just to do 10 runs for fork tests